javascript/javascript1/week3/saveNote.js

```diff
@@ -0,0 +1,53 @@
+
+// Save a note:
+
+const notes = [];
+
+function saveNote(content, id) {
+notes.push({content,id});
+
+  return notes;
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+console.log(notes);
+
+// Get a note:
+
+function getNote(id) {
+  for(let i=0; i<= notes.length; i++){
+    if( notes[i].id=== id || id===Number){
```

Comment:
`|| id===Number` thing is not required here

hint: in all cases below it will print false
```javascript
console.log(1 === Number)
const id  = 1
console.log(id  === Number)
```

To check if variable is number it's better to use
```typeof id === "number"```